MahApps.Metro/Microsoft.Windows.Shell/WindowChromeWorker.cs

```diff
@@ -933,17 +988,6 @@ private void _UpdateFrameState(bool force)
 
             if (force || frameState != _isGlassEnabled)
             {
-                if (SystemParameters.MinimizeAnimation && _chromeInfo.IgnoreTaskbarOnMaximize == false/* && _chromeInfo.UseNoneWindowStyle == false*/)
```

Comment:
why removing this? so it's not possible to change `IgnoreTaskbarOnMaximize` at runtime